index.html

```diff
@@ -0,0 +1,108 @@
+    <blockquote class="blockquote text-center quote">
+      <p class="mb-0 quote-text">"Every artist was first an amateur"</p>
+      <footer class="blockquote-footer">Ralph Waldo Emerson</footer>
+    </blockquote>    
+    <header class="container fluid">
+      <div class="row">
+        <nav class="col-sm-12">
+          <a class="btn btn-default navigation-button" href="#one" role="button">Biography</a>
+          <a class="btn btn-default navigation-button" href="#two" role="button">Art</a>
+          <a class="btn btn-default navigation-button" href="#three" role="button">Awards</a>
+          <a class="btn btn-default navigation-button" href="#images" role="button">Works</a>
+          <a class="btn btn-default btn-sm" href="https://www.facebook.com/">
+            <i class="fa fa-facebook fa-3x icon"></i>
+          </a>
+          <a class="btn btn-default btn-sm" href="https://twitter.com/?lang=pl">
+            <i class="fa fa-twitter fa-3x icon"></i>
+          </a>
+          <a class="btn btn-default btn-sm" href="https://www.instagram.com/">
+            <i class="fa fa-instagram fa-3x icon"></i>
+          </a>
+        </nav>
+      </div>
+    </header>
+    <div class="container">
+      <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+        <ol class="carousel-indicators">
+          <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+          <li data-target="#carousel-example-generic" data-slide-to="1"></li>
+          <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+          <li data-target="#carousel-example-generic" data-slide-to="3"></li>
+        </ol>
+        <div class="carousel-inner">
+          <div class="item active">
+            <img src="https://i.ytimg.com/vi/xMBnUtfrEgA/maxresdefault.jpg" alt="lala">            
+          </div>
+          <div class="item">
+            <img src="http://echogonewrong.com/wp-content/uploads/2018/03/rooster_gallery_022.jpg" alt="haha">
+          </div>
+          <div class="item">
+            <img src="http://www.roostergallery.eu/NEWEST/wp-content/uploads/2014/04/From-cycle-Iron-John-oil-on-canvas-15x215-cm-2011.jpg" alt="blabla">
+          </div>
+          <div class="item">
+            <img src="https://d32dm0rphc51dk.cloudfront.net/WalhpKixnabQ6gBRYsiUCg/larger.jpg" alt="fourth image">
+          </div>
+          <a class="left carousel-control" href="#carousel-example-generic" data-slide="prev">
+            <span class="icon-prev"></span>
+          </a>
+          <a class="right carousel-control" href="#carousel-example-generic" data-slide="next">
+            <span class="icon-next"></span>
+          </a>
+        </div>
+      </div>
+    </div>
+    <section class="jumbotron">
+      <div class="container main-space">
+        <h1>Kristina Ališauskaitė</h1>
+      </div>
+    </section>
+    <div class="main-information">
+      <section id="one">

```

Comment:
to co mowilismy, czyli zmien prosze z one, two, three na cos co ma jakies znaczenie, np bio, art, awards